### PR TITLE
Playlist

### PR DESCRIPTION
**Pull request by ark2002**
- All Playlist functionality added

Preview :

https://deploy-preview-8--astrotube.netlify.app/

recording

### CODE REVIEW

File: src/frontend/components/Navbar/Navbar.css [hunk 1]
```diff
@@ -5,19 +5,19 @@
     text-align: center;
     position: fixed;
     right: 0;
-    z-index: 2;
+    z-index: 9999;
```
Comment (MarkVed17) [inline src/frontend/components/Navbar/Navbar.css:8]: such a large z-index is not required.

**Review by kumsomi — APPROVED** [review]
- Playlist is working fine
- Liked the UI

**Review by MarkVed17 — APPROVED** [review]
- Everything functioning properly
- UI looks good
- Good to merge

**Review by chanchal-panpaliya — COMMENTED** [review]
I know you merged this branch. but please resolve the issue. ->
image

**Pull request merged**